frontend/src/scenes/session-recordings/player/inspector/components/ItemComment.tsx

```diff
@@ -0,0 +1,47 @@
+import { useActions } from 'kea'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { notebookPanelLogic } from 'scenes/notebooks/NotebookPanel/notebookPanelLogic'
+import { InspectorListItemComment } from 'scenes/session-recordings/player/inspector/playerInspectorLogic'
+
+export interface ItemCommentProps {
+    item: InspectorListItemComment
+    expanded: boolean
+    setExpanded: (expanded: boolean) => void
+}
+
+export function ItemComment({ item, expanded, setExpanded }: ItemCommentProps): JSX.Element {
+    const { selectNotebook } = useActions(notebookPanelLogic)
+
+    return (
+        <div data-attr="item-comment">
+            <LemonButton noPadding onClick={() => setExpanded(!expanded)} fullWidth className="font-normal">
+                {expanded ? (
+                    <div className="p-2 text-xs border-t w-full flex justify-end">
+                        <LemonButton
+                            type="secondary"
+                            onClick={(e) => {
+                                selectNotebook(item.data.notebookShortId)
+                                e.stopPropagation()
+                                e.preventDefault()
+                            }}
+                        >
+                            Continue in {item.data.notebookTitle}
```

Comment:
Does this truncate nicely? Conscious that notebook titles might be quite long

Reply:
I believe I fixed truncation in the inspector items 🙈